app/models/service_ansible_playbook.rb

```diff
@@ -46,6 +46,12 @@ def postprocess(action)
     delete_inventory(action) unless use_default_inventory?(hosts)
   end
 
+  def on_error(action)
```

Comment:
If this is only intended to be called from automate should we have a more descriptive method name?